samples/excel/89-preview-apis/chart-series.yaml

```diff
@@ -0,0 +1,171 @@
+order: 1
+id: excel-89-preview-apis-chart-series-
+name: Chart series
+description: 'Add, set, and delete a series in a chart.'
+host: EXCEL
+api_set:
+    ExcelAPI: 1.7
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-series").click(() => tryCatch(addSeries));
+        $("#delete-series").click(() => tryCatch(deleteSeries));
+        $("#add-series-value").click(() => tryCatch(setSeriesValue));
+
+        async function addSeries() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+
+                let seriesCollection = sheet.charts.getItemAt(0);
+                let rangeSelection = sheet.getRange("B1:B7");
+
+                // Add a series.
+                let newSeries = seriesCollection.series.add("New Series");
+                newSeries.setValues(rangeSelection);
+
+                await context.sync();
+            });
+        }
+
+        async function deleteSeries() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+
+                let seriesCollection = sheet.charts.getItemAt(0).series;
+                let series = seriesCollection.getItemAt(0);
+                // Delete the first series.
+                series.delete();
+
+                await context.sync();
+            });
+        }
+
+        async function setSeriesValue() {
```

Comment:
I'm not understanding this function/example at all, either by reading through the code and trying to match that up with the function name / button name, or by looking at the chart that it's producing (with numbers along both the x-axis and y-axis and legend labels "New Series" and "New Product".  What API functionality are we trying to showcase with this function?